src/containers/Ideas/Idea.js

```diff
@@ -358,102 +359,94 @@ export const Idea = (props) => {
           keyboardShouldPersistTaps='handled'
           onScroll={handleScroll}
         >
-          <Pressable
-            accessibilityRole="button"
-            onPress={isEditing && toggleEditing}
-            style={{
-              ...isEditing ? styles.pressableEditing : {}
-            }}
-            disabled={!isEditing}
-          >
```

Comment:
@khamui do you know why we use a pressable element which wraps the idea and comment list? it messes with the a11y so I removed it without any noticeable issues but not sure i'm missing something?

Reply:
@philli-m not sure, it seems to be something with when you are `editing` the Idea? Does everything work as expected if you are editing the idea?